manifest

```diff
@@ -101,6 +101,7 @@ export PACKAGES="\
 	podman \
 	pulsemixer \
 	python \
+	python-inotify-simple \
```

Comment:
This should be in a separate PR since affects the distribution. Isn't this package a dependency already?